content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -255,6 +256,169 @@ When the workloads are deployed, above, they send CSR Requests to Istiod which f
     $ kubectl delete ns bookinfo
     {{< /text >}}
 
+## Part 3: Using Custom CA with different signers for different workloads
+
+For this example, we use [open-source cert-manager](https://cert-manager.io).
+cert-manager has added [experimental Support for Kubernetes CertificateSigningRequests](https://cert-manager.io/docs/usage/kube-csr/) from v1.4.
+
+### Deploy Custom CA controller in the Kubernetes cluster
+
+1. Deploy cert-manager according to [installation doc](https://cert-manager.io/docs/installation/)
+   {{< warning >}}
+   Note Make sure to enable featureGate: --feature-gates=ExperimentalCertificateSigningRequestControllers=true
+   {{< /warning >}}
+
+1. Create three selfSigned clusterIssuers `istio-system` , `foo` and `bar` for cert-manager.
+
+### Export the clusterIssuer's root certificate
+
+    {{< text bash >}}
+    $ export istioca=$(kubectl get clusterissuers istio-system -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' |base64 -d)
+
+    $ export fooca=$(kubectl get clusterissuers foo -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' |base64 -d)
+
+    $ export barca=$(kubectl get clusterissuers bar -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' |base64 -d)
+    {{< /text >}}
+
+### Deploying Istio
+
+1. Deploy Istio on the cluster using `istioctl` with the following configuration.
+
+    {{< text bash >}}
+    $ cat <<EOF > ./istio.yaml
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    spec:
+      meshConfig:
+        defaultConfig:
+          proxyMetadata:
+            ISTIO_META_CERT_SIGNER: istio-system
+        caCertificates:
+        - pem: |
+          $istioca
+          certSigners:
+          - clusterissuers.cert-manager.io/istio-system
+        - pem: |
+          $fooca
+          certSigners:
+          - clusterissuers.cert-manager.io/foo
+        - pem: |
+          $barca
+          certSigners:
+          - clusterissuers.cert-manager.io/bar
+      components:
+        pilot:
+          k8s:
+            env:
+            - name: CERT_SIGNER_DOMAIN
+              value: clusterissuers.cert-manager.io
+            - name: EXTERNAL_CA
+              value: ISTIOD_RA_KUBERNETES_API
+            - name: PILOT_CERT_PROVIDER
+              value: k8s.io/clusterissuers.cert-manager.io/istio-system
+            overlays:
+              - kind: ClusterRole
+                name: istiod-clusterrole-istio-system
+                patches:
+                  - path: rules[-1]
+                    value: |
+                      apiGroups:
+                      - certificates.k8s.io
+                      resourceNames:
+                      - clusterissuers.cert-manager.io/*
```

Comment:
nit: this permission seems too wide? can we narrow it down to the three specific signers?